_posts/minutes/2016-10-07-minutes.markdown

```diff
@@ -1,4 +1,4 @@
-[LISTEN TO THIS MEETING](https://fccdl.in/iPvoj2r1R)
+&nbsp;&nbsp;&nbsp;Listen to this meeting here: [here](https://fccdl.in/iPvoj2r1R)

```

Comment:
![screen shot 2016-10-11 at 4 24 57 pm](https://cloud.githubusercontent.com/assets/7812814/19292212/58fbe6e6-8fcf-11e6-9273-08062d59067b.png)

The ordered list.